### PR TITLE
Handle fetch result with 400 response

### PR DESCRIPTION
**Pull request by tnguyenquy**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -201,22 +194,16 @@ export function initialize(clientSdkKey, flagKeys, user, specifiedOptions, platf
...
     let flag = flags[flagKey];
-    
     if (flag === undefined) {
         logger.error('Flag not found. No variation configuration available');
         return {};
     } else { 
         let variation = flag.result;
-        
         if (!variation) {
-            logger.error(`Variation key ${variationKey} not found!. No variation configuration available`);
```
Comment (tnguyenquy) [inline src/index.js:219]: Change message, because variationKey doesn't exist

File: src/Requestor.js [hunk 2]
```diff
@@ -63,7 +56,19 @@ export default function Requestor(platform, options, environment) {
             const message = messages.invalidContentType(result.header('content-type') || '');
             return Promise.reject(new errors.ULFlagFetchError(message));
           }
-        } else {
+        }
+        else if (result.status === 400) {
```
Comment (tnguyenquy) [inline src/Requestor.js:60]: I want to test this change but don't know how. I tried npm link, and run a test script using node but I got error at this line `if (window.XMLHttpRequest) {` in ulClient.js. The reason is that `window` is undefined in for node.

File: src/EventProcessor.js [hunk 3]
```diff
@@ -50,9 +46,6 @@ export default function EventProcessor(
     if (disabled) {
       return;
     }
-    let addFullEvent = false;
-    let addDebugEvent = false;
```
Comment (umermansoor) [inline src/EventProcessor.js:54]: I'm assuming these were unused?

File: src/Requestor.js (hunk 2 above)
Comment (umermansoor) [reply src/Requestor.js:60]: You'd have to test this in browser. I created an index.html for it. Let me find and ping you on Slack.

**Review by umermansoor — APPROVED** [review]

**Pull request merged**